File: Logic_HandsOn/Java/Problems on Array,Pattern etc/Program73.java
////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
//
//	Program:
//		Accept N numbers from user check whether that numbers contains 11 in it or not.	
//
////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////


import java.util.*;

class Solution
{
	public int iSize,arr[],iCnt;

	public Solution(int iNo)
	{
		this.iCnt = 0;
		this.iSize = iNo;
		arr = new int[iSize];
	}

	public void Accept()
	{
		Scanner sObj = new Scanner(System.in);

		System.out.println("Enter "+arr.length+" elements: ");
		for(iCnt=0;iCnt<arr.length;iCnt++)
		{
			System.out.print("Enter "+ (iCnt+1) + "element: ");
			arr[iCnt] = sObj.nextInt();
		}
	}	

	public void CountEleven()
	{	
		
		for(iCnt=0;iCnt<arr.length;iCnt++)
		{
			if(arr[iCnt]==11)
			{
				System.out.println("11 is present");
				System.exit(0);
			}
			
		}
		System.out.println("11 is absent");
	}


}

class Program
{
	public static void main(String[] args)
	{
		Scanner sObj = new Scanner(System.in);

		System.out.println("Enter number of elements:");
		int iLength = sObj.nextInt();

		Solution obj = new Solution(iLength);
		obj.Accept();
		obj.CountEleven();
	}
}

/*
OUTPUT:
Enter number of elements:
6
Enter 6 elements: 
Enter 1element: 85
Enter 2element: 66
Enter 3element: 11
Enter 4element: 80
Enter 5element: 93
Enter 6element: 88
11 is present

*/

File: Logic_HandsOn/C++/Problems on Digits,String,Array,Pointers etc/program73.cpp
////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
//
//	Program:
//		Accept N numbers from user check whether that numbers contains 11 in it or not.	
//
////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////


#include<iostream>
using namespace std;

class Solution
{
	private:
		int *iArr;
		int iSize,iCnt;

	public:
		Solution(int iNo=0,int iCnt=0)
		{
			this->iSize=iNo;
			iArr = new int[iSize];
		}

		void Accept()
		{
			cout<<"Enter "<<iSize<<" elements:\n";
			for(iCnt=0;iCnt<iSize;iCnt++)
			{
				cout<<"Enter "<<iCnt+1<<" element: ";
				cin>>iArr[iCnt];
			}
		}
		
		bool CountEleven()
		{
			for(iCnt=0;iCnt<iSize;iCnt++)
			{
				if(iArr[iCnt]==11)
				{
					return true;
				}
				
			}

		}

		~Solution()
		{
			delete []iArr;
		}

};

int main()
{
	int iLength=0;
	bool bRet=false;

	cout<<"Enter number of elements: ";
	cin>>iLength;

	Solution *dObj = new Solution(iLength);

	dObj->Accept();
	bRet = dObj->CountEleven();

	if(bRet==true)
	{
		cout<<"11 is present\n";
	}
	else
	{
		cout<<"11 is absent\n";
	
	}

	delete dObj;
	return 0;
}

/*
OUTPUT:

Enter number of elements: 6
Enter 6 elements:
Enter 1 element: 85
Enter 2 element: 66
Enter 3 element: 11
Enter 4 element: 80
Enter 5 element: 93
Enter 6 element: 88
11 is present

*/
